gravitee-apim-console-webui/src/management/api/apis.module.ts

```diff
@@ -1355,6 +1356,20 @@ const states: Ng2StateDeclaration[] = [
     },
     component: ApiRuntimeLogsComponent,
   },
+  {
+    name: 'management.apis.ng.runtimeLogs-messages',
+    url: '/runtime-logs/:requestId',
```

Comment:
are you sure about this url, it does not match what I pushed 🤔

Reply:
As discussed it's the link to go to the page ;)